<launch>
    <node name="ugv_control_node" pkg="sunray_ugv_control" type="ugv_control_node" output="screen">
        <param name="ugv_id" value="1" />
        <param name="flag_printf" value="true" />
        <param name="enable_rviz" value="true" />
        <!-- zidingyi odom  -->
        <param name="location_source" value="2" />
        <param name="odom_topic" value="/odom" />
        <param name="vel_topic" value="/cmd_vel" />
        <!-- 控制参数 -->
        <param name="ugv_control_param/Kp_xy" value="2.0" />
        <param name="ugv_control_param/Kp_yaw" value="3.14" />
        <param name="ugv_control_param/max_vel_xy" value="1.5" />
        <param name="ugv_control_param/max_vel_yaw" value="6" />
        <!-- 地理围栏参数 -->
        <param name="ugv_geo_fence/max_x" value="100.0" />
        <param name="ugv_geo_fence/min_x" value="-100.0" />
        <param name="ugv_geo_fence/max_y" value="100.0" />
        <param name="ugv_geo_fence/min_y" value="-100.0" />
        <!-- 地图参数 -->
        <param name="map/resolution" value="0.4" />
        <param name="map/inflate" value="0.3" />
        <!-- 仿真模式参数 -->
        <param name="simulation_mode" value="true" />
        <!-- 雷达话题参数 -->
        <param name="laser_topic" value="/sunray/ugv1/scan" />
        <!-- 地图相关话题参数 -->
        <param name="octomap_topic" value="/octomap_tree" />
        <param name="occupancy_topic" value="/occupancy_grid" />
    </node>
</launch>
